cpp/src/parquet/arrow/arrow-reader-writer-test.cc

```diff
@@ -140,6 +140,69 @@ LogicalType::type get_logical_type(const ::DataType& type) {
   return LogicalType::NONE;
 }
 
+std::shared_ptr<::arrow::DataType> get_arrow_type(const PrimitiveNode& p) {
```

Comment:
Isn't this https://github.com/apache/arrow/blob/master/cpp/src/parquet/arrow/schema.cc#L168?